projects/js-packages/number-formatters/package.json

```
@@ -0,0 +1,38 @@
+{
+	"name": "@automattic/jetpack-number-formatters",
+	"version": "0.1.0",
+	"description": "Number formatting utilities",
+	"author": "Automattic",
+	"license": "GPL-2.0-or-later",
+	"homepage": "https://github.com/Automattic/jetpack/tree/HEAD/projects/js-packages/number-formatters/#readme",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git",
+		"directory": "projects/js-packages/number-formatters"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/labels/[JS Package] Number Formatters"
+	},
+	"dependencies": {
+		"@babel/runtime": "^7"
+	},
+	"devDependencies": {
+		"@babel/core": "7.26.0",
```

The rest of the monorepo is on 7.26.10.
```suggestion
		"@babel/core": "7.26.10",
```